net-dns/https_dns_proxy/https_dns_proxy-9999.ebuild

```diff
@@ -20,12 +20,25 @@ HOMEPAGE="https://github.com/aarond10/https_dns_proxy"
 LICENSE="MIT"
 SLOT="0"
 
-DEPEND="dev-libs/libev
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-libs/libev
 	net-dns/c-ares
 	net-misc/curl[http2,ssl]"
-RDEPEND="${DEPEND}"
+DEPEND="${RDEPEND}
+	test? ( dev-cpp/gtest )"
```

Comment:
```
>>> Test phase: net-dns/https_dns_proxy-0_pre20200925
debug: cmake_src_test: entering function, parameters: 
 * Working in BUILD_DIR: "/var/tmp/portage/net-dns/https_dns_proxy-0_pre20200925/work/https_dns_proxy-0_pre20200925_build"
ctest -j 30 --test-load 999
Test project /var/tmp/portage/net-dns/https_dns_proxy-0_pre20200925/work/https_dns_proxy-0_pre20200925_build
No tests were found!!!
 * Tests succeeded.
>>> Completed testing net-dns/https_dns_proxy-0_pre20200925
```
:thinking:

Reply:
@SoapGentoo Whoops, sorry, that was a long day :/
There are actually no test implementations right now, yet cmake checks for gtest. I'm on it now, will disable redundant package checks :)

Reply:
Done